src/root.jsx

```diff
@@ -188,8 +188,12 @@ class Root extends React.Component {
         return (
             <div>
                 <p>Meeting date <quip.apps.ui.RichTextBox record={date} /></p>
-                <p>toastmaster <quip.apps.ui.RichTextBox record={toastmaster} /></p>
-                <p>jokemaster <quip.apps.ui.RichTextBox record={jokemaster} /></p>
+                <p>Toastmaster <quip.apps.ui.RichTextBox record={toastmaster} /></p>
+                <p>Grammarian <quip.apps.ui.RichTextBox record={grammarian} /></p>
+                <p>Timer <quip.apps.ui.RichTextBox record={timer} /></p>
+                <p>Ah Counter <quip.apps.ui.RichTextBox record={ahCounter} /></p>
+                <p>Jokemaster <quip.apps.ui.RichTextBox record={jokemaster} /></p>
```

Comment:
@borncrusader did you mean to rearrange the role order? Seems to me Jokemaster would come immediately after Toastmaster to be consistent with the actual meeting flow.